src/common.rs

```diff
@@ -83,19 +83,27 @@ pub(crate) fn fix_symbol_name<'a>(name: &'a Name<'a>) -> Name<'a> {
     }
     let fixed = LLVM_NNN.replace(name.as_str(), "");
```

Comment:
Do those suffices appear in mangled or demangled names? I'm considering to strip those out in symbolic already.

Reply:
Before demangling, see this change: https://github.com/mozilla/dump_syms/commit/4d57c02a3b9659df2e3804848390fe8a2a640a59

Reply:
Got it, thanks. I would not mind adding this to symbolic then, but only for `Function::name` and not for the raw symbol table. Would that work for your case?

Reply:
Maybe it could be interesting to parameterize the demangler with 2 traits for pre and post hooks to run on the name.

Reply:
I think that could be worth it. Although, looking at #130, I think that this can be a concern outside of the demangler. 

FWIW, we have a similar issue at Sentry, where we need to post-process demangled names so we can consider them for grouping. We even go further and strip modifiers, all suffices, and even generics in some cases. We could add a utility that takes a demangled name (important for Breakpad symbols) and spits out a canonicalized, simplified form of it.

@calixteman would it be enough if we add a flag to disable demangling of suffices and then pass this through to cpp_demangle (and potentially others)?